docs/concepts/Auditing-Packages.md

```diff
@@ -87,8 +87,15 @@ We recommend that audit is configured at a repository level.
 
 ### Excluding advisories
 
-There is no support for excluding individual advisories at this time.
-You can use `<NoWarn>` to suppress `NU1901`-`NU1904` warnings or use the `<NuGetAuditLevel>` functionality to ensure your audit reports are useful to your workflow.
+You can choose to exclude specific advisories from the audit report by adding a new `NuGetAuditSuppress` MSBuild item for each advisory. Define a `NuGetAuditSuppress` item with the `Include=` metadata set to the advisory URL you wish to suppress.
+
+```xml
+    <NuGetAuditSuppress Include="https://github.com/advisories/XXXX" />
```

Comment:
```suggestion
<ItemGroup>
    <NuGetAuditSuppress Include="https://github.com/advisories/XXXX" />
</ItemGroup>
```

All other NuGetAudit settings are properties, not items, and I think a lot of customers won't be familiar enough with MSBuild syntax to notice the `Include=` means it's an item.  I think adding the `<ItemGroup>` parent increases the chance customers will be successful.

Reply:
That's a good point. I've added that in